packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.base.tsx

```diff
@@ -261,17 +261,12 @@ export class DatePickerBase extends BaseComponent<IDatePickerProps, IDatePickerS
   }
 
   private _onSelectDate = (date: Date): void => {
-    const { formatDate, onSelectDate } = this.props;
+    const { onSelectDate } = this.props;
 
     if (this.props.calendarProps && this.props.calendarProps.onSelectDate) {
       this.props.calendarProps.onSelectDate(date);
     }
 
-    this.setState({
-      selectedDate: date,
-      formattedDate: formatDate && date ? formatDate(date) : ''
-    });
```

Comment:
Are you sure this should be removed entirely? What about uncontrolled scenarios? It appears [this has broken the existing uncontrolled examples](http://fabricweb.z5.web.core.windows.net/pr/refs/pull/6590/merge/#/examples/datepicker).

Reply:
I will take care of this.